gson/src/main/java/com/google/gson/stream/JsonReader.java

```diff
@@ -321,6 +319,18 @@ public JsonReader(Reader in) {
    *       {@code :}.
    *   <li>Name/value pairs separated by {@code ;} instead of {@code ,}.
    * </ul>
+   *
+   * <p>Note: Even in strict mode there are slight derivations from the JSON
```

Comment:
Maybe this should clarify that these derivations are not guaranteed behavior and might change in the future?